pkg/sfu/videolayerselector/dependencydescriptor.go

```diff
@@ -252,6 +257,7 @@ func (d *DependencyDescriptor) Select(extPkt *buffer.ExtPacket, _layer int32) (r
 		if d.needsDecodeTargetBitmask {
 			d.needsDecodeTargetBitmask = false
 
+			// DD-TODO : should use currentLayer instead of targetLayer, and update it every time we switch layer(not only when we switch targetLayer)
```

Comment:
this might be a chicken and egg issue on start up? We will not have a current initially till we start streaming. How does using `targetLayer` instead of `currentLayer` affect functionality?

Reply:
Just found the issue but have not though a solution for it, so left the comments here. The issue is browser always try to decode the highest `ActiveDecodeTarget` in my test before, which makes sense as it makes stable resolution and frame rates, but if the highest decode target is broken, it will send PLI to resync. So we should assure that the bitmask is consistent with the current layer to assure the current decode target is intact.